test/webhook-apicoverage/view/html_display.go

```diff
@@ -17,115 +17,53 @@ limitations under the License.
 package view
 
 import (
-	"fmt"
 	"strings"
+	"text/template"
```

Comment:
Use "html/template" instead of "text/template" for html content. 

_"Package template (html/template) implements data-driven templates for generating HTML output safe against code injection. It provides the same interface as package text/template and should be used instead of text/template whenever the output is HTML."_ 
source: https://golang.org/pkg/html/template/

Reply:
Ack will update.

Reply:
Done